app/templates/admin/main.html

```diff
@@ -72,28 +72,28 @@ <h2 class="text-center">Manage {% if current_user.is_super %}Agencies and{% else
             {% endif %}
             <div class="col-sm-{% if current_user.is_super %}1{% else %}2{% endif %} text-right">
                 <button type="button" class="btn btn-danger btn-sm" id="remove">
-                     <span class="glyphicon glyphicon-trash" aria-hidden="true"></span>
+                    <span class="glyphicon glyphicon-trash" aria-hidden="true"></span>
                 </button>
             </div>
         </div>
         {% for user in users %}
             <div class="row user-info">
```

Comment:
Can add `break-word` class here to the rest of the child divs have `style="word-wrap: break-word"`